src/backend/data/post.js

```diff
@@ -148,7 +149,16 @@ class Post {
       article.date = article.pubdate;
     }
 
+    // All the Youtube feed return an array off html so we will need to convert it to a string so as to process and sanitize it
+    if (Array.isArray(article.content)) {
+      article.content = article.content.join(' ');
+    }
+
+    // Wrap an <a> tag on any link inside our content
+    article.content = linkifyHtml(article.content);
```

Comment:
Can this `throw`?  Check to see if it's always safe to call without a `try/catch`

Reply:
For reference, this appears to be the function called here if I'm correct, and I don't see any case of an exception being thrown here
https://github.com/Hypercontext/linkifyjs/blob/main/packages/linkifyjs/src/linkify-html.js#L16

Reply:
From what I understand,   
https://github.com/Hypercontext/linkifyjs/blob/main/packages/linkifyjs/src/linkify-html.js#L16
It will not `throw` and we did manage our parameter to become a `string` here. The reason for that was when I do `debugging` it just did not stop the software even when the parameter passing into it was wrong.
https://github.com/Seneca-CDOT/telescope/blob/3bdcc3a1f03a3abcdb277f56439e1498e12afcb1/src/backend/data/post.js#L153-L154